app/src/main/res/layout/activity_product_page.xml

```diff
@@ -2,147 +2,201 @@
-        app:layout_constraintHeight_percent="0.04"
-        android:text="hello"
-        android:textSize="18sp"
-        app:layout_constraintTop_toBottomOf="@id/product_name_guide"
-        app:layout_constraintLeft_toLeftOf="@id/product_info_left_guide"/>
-
-    <TextView
-        android:id="@+id/description"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        app:layout_constraintBottom_toBottomOf="@+id/name"
-        app:layout_constraintLeft_toLeftOf="@id/product_info_left_guide"/>
-
-
-    <androidx.constraintlayout.widget.Guideline
-        android:id="@+id/product_info_guide"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:orientation="horizontal"
-        app:layout_constraintGuide_percent="0.5" />
-
-    <androidx.constraintlayout.widget.Guideline
-        android:id="@+id/product_info_left_guide"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:orientation="vertical"
-        app:layout_constraintGuide_percent="0.04" />
-
-    <LinearLayout
-        android:layout_width="wrap_content"
+        app:layout_constraintBottom_toTopOf="@id/product_page_bottom_area"
+        app:layout_constraintTop_toBottomOf="@id/product_page_back_area">
+        <ScrollView
+            android:id="@+id/product_page_scrollview"
+            android:layout_width="fill_parent"
+            android:layout_height="fill_parent">
+
+            <TableLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:background="@color/backgroundColor"
+                android:layoutDirection="ltr"
+                android:orientation="vertical">
+
+
+                <TableRow
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:gravity="center">
+
+                    <ImageView
+                        android:id="@+id/prpImagePlaceholder"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:adjustViewBounds="true"
+                        android:background="@null"
+                        android:gravity="center"
+                        android:paddingTop="@dimen/productPageSectionPaddingTop"
+                        android:scaleType="fitCenter" />
+                </TableRow>
+
+                <TableRow
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content">
+
+                    <TextView
+                        android:id="@+id/textviewProductPageTitle"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:fontFamily="sans-serif"
```

Comment:
Maybe I am wrong, but currently, font-family is defined in res->values->styles so this is not necessary?

Reply:
You are right

Reply:
as I mentioned the product page is just a skeleton, it will be much more clean in the next few days. Still it's important for me to push it for integration